roles/lib_utils/callback_plugins/aa_version_requirement.py

```diff
@@ -22,7 +22,7 @@ def display(*args, **kwargs):
 
 # Set to minimum required Ansible version
 REQUIRED_VERSION = '2.5.7'
-DESCRIPTION = "Supported versions: %s or newer but less than 2.8.0" % REQUIRED_VERSION
+DESCRIPTION = "Supported versions: %s or newer" % REQUIRED_VERSION
```

Comment:
Maybe its worth restricting 2.9.0 already, as it may have breaking changes?

Reply:
I'll add it if testing 2.9 RCs shows major problems.  I hope to be able to have anything fixed before 2.9 shipped.